prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -62,6 +62,33 @@ postsubmits:
         - boskos
         - deploy
 
+  - name: post-test-infra-push-prowbazel
+    cluster: test-infra-trusted
+    run_if_changed: '^docker/prowbazel/(Dockerfile|entrypoint)$'
```

Comment:
If you look at the last few prowbazel changes, you'll see that those files are not modified at all.

This should be triggered on changes to `docker/prowbazel/Makefile`, which bumps the version.

Reply:
Okay, makes sense. Updated.